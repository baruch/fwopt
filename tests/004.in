-A INPUT --dport 80 -j ACCEPT
