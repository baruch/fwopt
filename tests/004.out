Setting dest port but protocol is not udp nor tcp
Error: Failed setting rule
-A INPUT --dport 80 -j ACCEPT
                 ^^^
Failed parsing input
